fire-hose-management-frontend/src/components/Order/OrderForm.jsx

```diff
@@ -126,13 +123,31 @@ const OrderForm = () => {
   const handleSubmit = ( event ) => {
     event.preventDefault()
 
-    let orderDto = {};
-    orderDto.department = department;
-    orderDto.devices = showDevices.filter( device => device.count > 0 );
-    orderDto.forename = forename;
-    orderDto.surname = surname;
-    orderDto.notes = notes
-
+    let orderDto = {
+      department: {
+        id: department.id,
+        departmentName: department.departmentName,
+        contact: {
+          firstname: department.contact.firstname,
+          lastname: department.contact.lastname,
+          gender: department.contact.gender,
+          mail: department.contact.mail
+        },
+        street: department.street,
+        houseNumber: department.houseNumber,
+        location: department.location,
+        postalCode: department.postalCode,
+        country: department.country,
+        registered: department.registered,
+        sendConfirmationMail: department.sendConfirmationMail
+      },
+      contact: null,
+      devices: showDevices.filter( device => device.count > 0 ),
+      firstname: firstname,
+      lastname: lastname,
+      notes: (notes !== "") ? notes : null
```

Comment:
Codacy has a fix for the issue: [Replace `(notes·!==·"")·?·notes·:·null` with `notes·!==·""·?·notes·:·null,`](https://app.codacy.com/gh/StitZle/fire-hose-complete/pullRequest?prid=9542260)
```suggestion
      notes: notes !== "" ? notes : null,
```